I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 1331, in script call
    call memoria_game1
  File "game/script.rpy", line 1294, in script
    jump turns_loop
ScriptError: could not find label 'turns_loop'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 1331, in script call
    call memoria_game1
  File "game/script.rpy", line 1294, in script
    jump turns_loop
  File "C:\Users\user\Downloads\renpy-8.0.3-sdk\renpy\ast.py", line 1975, in execute
    rv = renpy.game.script.lookup(target)
  File "C:\Users\user\Downloads\renpy-8.0.3-sdk\renpy\script.py", line 927, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'turns_loop'.

Windows-10-10.0.19045 AMD64
Ren'Py 8.0.3.22090809
Sabby Dreams 1.0
Mon May 22 03:23:40 2023
